src/Data/List/Relation/Ternary/Appending/Propositional.agda

```diff
@@ -0,0 +1,41 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Appending of lists using propositional equality
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.List.Relation.Ternary.Appending.Propositional {a} {A : Set a} where
+
+open import Data.List.Base as List using (List; []; _∷_)
+import Data.List.Properties as Listₚ
+import Data.List.Relation.Binary.Pointwise as Pw
+import Data.List.Relation.Ternary.Appending.Setoid as General
+import Data.List.Relation.Ternary.Appending.Properties as Appendingₚ
+open import Data.Product using (_,_)
+open import Relation.Binary.PropositionalEquality
+  using (_≡_; setoid; refl; trans; cong; cong₂; module ≡-Reasoning)
+
+------------------------------------------------------------------------
+-- Re-export the basic combinators
+
+open General hiding (Appending; _++_; break) public
```

Comment:
Style guide says that `public` should go before `using`/`hiding`/`renaming`. Same for other files below.